Publish initial draft of the emnify Portal user guide (#142)
emnify/product-docs: docs/portal/organization/workspaces.md

---
title: Workspaces
description: Organizations with complex business structures or diverse product lines can benefit from Workspaces in the emnify Portal
slug: /workspaces
---

# Workspaces <span className="theme-doc-version-badge badge badge--primary beta">Beta</span>

> This feature is only available to [**Pro** plan](https://portal.emnify.com/organisation-settings/subscription) subscribers.

Reviewer: ```suggestion
This feature comes with the [**Pro** plan](https://portal.emnify.com/organisation-settings/subscription) subscribers.
```
